tests/rpmbuild.at

```diff
@@ -995,6 +995,25 @@ runroot rpm -qp --requires /build/RPMS/noarch/shebang-0.1-1.noarch.rpm|grep -v ^
 [])
 RPMTEST_CLEANUP
 
+AT_SETUP([Local dependency generator])
+AT_KEYWORDS([build])
+RPMTEST_CHECK([
+RPMDB_INIT
+
+runroot rpmbuild -bb --quiet \
+		--define '__local_file_attrs my_test_attr' \
```

Comment:
If this covered multiple generators to demonstrate that something like this is supported, that would be even better